solr/packaging/test/test_create_collection.bats

```diff
@@ -18,12 +18,12 @@
 load bats_helper
 
 setup_file() {
-  common_setup
+  common_clean_setup
   solr start -c
 
   local source_configset_dir="$SOLR_TIP/server/solr/configsets/sample_techproducts_configs"
   test -d $source_configset_dir
-  cp -r $source_configset_dir "$BATS_TMPDIR/config"
+  cp -r "${source_configset_dir}" "$BATS_FILE_TMPDIR/config"
```

Comment:
Since this is only used by one test, can we move it to that test? Then I think we would use `BATS_TEST_TMPDIR`